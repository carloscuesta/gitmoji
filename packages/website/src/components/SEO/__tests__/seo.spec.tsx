import renderer from 'react-test-renderer'

import SEO from '../index'
import * as stubs from './stubs'

jest.mock('next/head', () => 'Head')

describe('SEO', () => {
  it('should render the component with pageTitle and pageUrl', () => {
    const wrapper = renderer.create(<SEO {...stubs.props} />)
    expect(wrapper).toMatchSnapshot()
  })

  it('should render the component', () => {
    const wrapper = renderer.create(<SEO />)
    expect(wrapper).toMatchSnapshot()
  })
})
