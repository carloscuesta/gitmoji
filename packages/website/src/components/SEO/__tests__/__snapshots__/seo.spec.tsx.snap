// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEO should render the component 1`] = `
<Head>
  <title>
    gitmoji | An emoji guide for your commit messages
  </title>
  <link
    href="https://gitmoji.dev"
    rel="canonical"
  />
  <meta
    content="Carlos Cuesta"
    name="author"
  />
  <meta
    content="Gitmoji is an emoji guide for your commit messages. Aims to be a standarization cheatshee /t for using emojis on GitHub's commit messages."
    name="description"
  />
  <meta
    content="width=device-width, initial-scale=1.0"
    name="viewport"
  />
  <meta
    content="summary"
    name="twitter:card"
  />
  <meta
    content="gitmoji"
    name="twitter:title"
  />
  <meta
    content="An emoji guide for your commit messages."
    name="twitter:description"
  />
  <meta
    content="https://gitmoji.dev/static/gitmoji.gif"
    name="twitter:image"
  />
  <meta
    content="@crloscuesta"
    name="twitter:creator"
  />
  <meta
    content="https://gitmoji.dev"
    name="twitter:url"
  />
  <meta
    content="gitmoji"
    property="og:title"
  />
  <meta
    content="An emoji guide for your commit messages."
    name="og:description"
  />
  <meta
    content="https://gitmoji.dev/static/gitmoji.gif"
    property="og:image"
  />
  <meta
    content="https://gitmoji.dev"
    name="og:url"
  />
  <meta
    content="index, follow"
    name="robots"
  />
  <link
    href="/static/apple-icon-57x57.png"
    rel="apple-touch-icon"
    sizes="57x57"
  />
  <link
    href="/static/apple-icon-60x60.png"
    rel="apple-touch-icon"
    sizes="60x60"
  />
  <link
    href="/static/apple-icon-72x72.png"
    rel="apple-touch-icon"
    sizes="72x72"
  />
  <link
    href="/static/apple-icon-76x76.png"
    rel="apple-touch-icon"
    sizes="76x76"
  />
  <link
    href="/static/apple-icon-114x114.png"
    rel="apple-touch-icon"
    sizes="114x114"
  />
  <link
    href="/static/apple-icon-120x120.png"
    rel="apple-touch-icon"
    sizes="120x120"
  />
  <link
    href="/static/apple-icon-144x144.png"
    rel="apple-touch-icon"
    sizes="144x144"
  />
  <link
    href="/static/apple-icon-152x152.png"
    rel="apple-touch-icon"
    sizes="152x152"
  />
  <link
    href="/static/apple-icon-180x180.png"
    rel="apple-touch-icon"
    sizes="180x180"
  />
  <link
    href="/static/android-icon-192x192.png"
    rel="icon"
    sizes="192x192"
    type="image/png"
  />
  <link
    href="/static/favicon-32x32.png"
    rel="icon"
    sizes="32x32"
    type="image/png"
  />
  <link
    href="/static/favicon-96x96.png"
    rel="icon"
    sizes="96x96"
    type="image/png"
  />
  <link
    href="/static/favicon-16x16.png"
    rel="icon"
    sizes="16x16"
    type="image/png"
  />
  <link
    href="/static/manifest.json"
    rel="manifest"
  />
  <meta
    content="#FFDD67"
    name="msapplication-TileColor"
  />
  <meta
    content="/ms-icon-144x144.png"
    name="msapplication-TileImage"
  />
  <meta
    content="#FFDD67"
    name="theme-color"
  />
  <meta
    content="78vmlhi_erc-UGybxcGwHyiUtf04wzYExTLa-4LoWio"
    name="google-site-verification"
  />
  <link
    href="/static/opensearchdescription.xml"
    rel="search"
    type="application/opensearchdescription+xml"
  />
</Head>
`;

exports[`SEO should render the component with pageTitle and pageUrl 1`] = `
<Head>
  <title>
    gitmoji | About | An emoji guide for your commit messages
  </title>
  <link
    href="https://gitmoji.dev/about"
    rel="canonical"
  />
  <meta
    content="Carlos Cuesta"
    name="author"
  />
  <meta
    content="Gitmoji is an emoji guide for your commit messages. Aims to be a standarization cheatshee /t for using emojis on GitHub's commit messages."
    name="description"
  />
  <meta
    content="width=device-width, initial-scale=1.0"
    name="viewport"
  />
  <meta
    content="summary"
    name="twitter:card"
  />
  <meta
    content="gitmoji"
    name="twitter:title"
  />
  <meta
    content="An emoji guide for your commit messages."
    name="twitter:description"
  />
  <meta
    content="https://gitmoji.dev/static/gitmoji.gif"
    name="twitter:image"
  />
  <meta
    content="@crloscuesta"
    name="twitter:creator"
  />
  <meta
    content="https://gitmoji.dev"
    name="twitter:url"
  />
  <meta
    content="gitmoji"
    property="og:title"
  />
  <meta
    content="An emoji guide for your commit messages."
    name="og:description"
  />
  <meta
    content="https://gitmoji.dev/static/gitmoji.gif"
    property="og:image"
  />
  <meta
    content="https://gitmoji.dev"
    name="og:url"
  />
  <meta
    content="index, follow"
    name="robots"
  />
  <link
    href="/static/apple-icon-57x57.png"
    rel="apple-touch-icon"
    sizes="57x57"
  />
  <link
    href="/static/apple-icon-60x60.png"
    rel="apple-touch-icon"
    sizes="60x60"
  />
  <link
    href="/static/apple-icon-72x72.png"
    rel="apple-touch-icon"
    sizes="72x72"
  />
  <link
    href="/static/apple-icon-76x76.png"
    rel="apple-touch-icon"
    sizes="76x76"
  />
  <link
    href="/static/apple-icon-114x114.png"
    rel="apple-touch-icon"
    sizes="114x114"
  />
  <link
    href="/static/apple-icon-120x120.png"
    rel="apple-touch-icon"
    sizes="120x120"
  />
  <link
    href="/static/apple-icon-144x144.png"
    rel="apple-touch-icon"
    sizes="144x144"
  />
  <link
    href="/static/apple-icon-152x152.png"
    rel="apple-touch-icon"
    sizes="152x152"
  />
  <link
    href="/static/apple-icon-180x180.png"
    rel="apple-touch-icon"
    sizes="180x180"
  />
  <link
    href="/static/android-icon-192x192.png"
    rel="icon"
    sizes="192x192"
    type="image/png"
  />
  <link
    href="/static/favicon-32x32.png"
    rel="icon"
    sizes="32x32"
    type="image/png"
  />
  <link
    href="/static/favicon-96x96.png"
    rel="icon"
    sizes="96x96"
    type="image/png"
  />
  <link
    href="/static/favicon-16x16.png"
    rel="icon"
    sizes="16x16"
    type="image/png"
  />
  <link
    href="/static/manifest.json"
    rel="manifest"
  />
  <meta
    content="#FFDD67"
    name="msapplication-TileColor"
  />
  <meta
    content="/ms-icon-144x144.png"
    name="msapplication-TileImage"
  />
  <meta
    content="#FFDD67"
    name="theme-color"
  />
  <meta
    content="78vmlhi_erc-UGybxcGwHyiUtf04wzYExTLa-4LoWio"
    name="google-site-verification"
  />
  <link
    href="/static/opensearchdescription.xml"
    rel="search"
    type="application/opensearchdescription+xml"
  />
</Head>
`;
